cgroup_test.go

```diff
@@ -31,8 +31,8 @@ func TestGetAvailableCpusetList(t *testing.T) {
 		{"0-3", "0-3"},
 		{"0-3,4", fakeGuestCpuset},
 		{"0-4", fakeGuestCpuset},
-		{"1", fakeGuestCpuset},
-		{"1,3", "2,13"},
+		{"1", "1"},
+		{"1,3", "1,3"},
```

Comment:
Can't you fix this directly on the first commit introducing those tests?

Reply:
I reorganized the commits